packages/n/nvidia-glx-driver/files/nvidia-current.metainfo.xml

```diff
@@ -1,8 +1,8 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <component type="driver">
   <id>nvidia-glx-driver-current</id>
-  <name>NVIDIA Linux Graphics Driver (Current Kernel)</name>
-  <summary>Accelerated Linux Graphics Driver for the -current kernel</summary>
+  <name>NVIDIA Graphics Driver (Current Kernel)</name>
```

Comment:
The icon path here still specifies the old pixmap path. According to the [specification](https://freedesktop.org/software/appstream/docs/chap-CatalogData.html#tag-ct-icon) it looks like the correct way to have the icon is to have:
```
<icon type="stock">nvidia</icon>
```
This will allow the icon to be overridden at runtime by user themes (Papirus ships a nvidia icon for example).

Reply:
This feedback applies to `nvidia-lts.metainfo.xml` too.